Packages/com.unity.inputsystem/Tests/InputSystem/Plugins/PS4Tests.cs

```diff
@@ -0,0 +1,105 @@
+using UnityEngine.Experimental.Input;
+using UnityEngine.Experimental.Input.LowLevel;
+using UnityEngine.Experimental.Input.Plugins.DualShock;
+using UnityEngine.Experimental.Input.Plugins.DualShock.LowLevel;
+using UnityEngine.Experimental.Input.Processors;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.Experimental.Input.Layouts;
+using UnityEngine.TestTools.Utils;
+
+
+public class PS4Tests : InputTestFixture
```

Comment:
As part of moving PS4 stuff to its own plugin, please move PS4-specific tests over from DualShockSupport.